docs/api_autopilot.md

```diff
@@ -0,0 +1,56 @@
+# API autopilot
+
+Kusk gateway supports API auto deployment. 
+
+Any REST API will typically consist of a Kubernetes pod(s) that is running the server and a Kubernetes service pointed to it.
+
+## It just works
```

Comment:
I think immediately in this section we should show how to use the annotation rather than putting that in the under the hood section. We can explain in details how it works later but for pure usage, all the info should be here.

Reply:
I meant it to look like TL;DR thing

Reply:
I agree with Kyle, 

I would add a 
Add this to your service 
```
...
metada: 
  annotation
... 

and apply the change.

doesn't occupy a lot of space and gives a good TLDR visual